readme.md

```diff
@@ -39,8 +39,14 @@ A simple tweet processor for automatically and intelligently sorting tweets rela
 
     `poetry run python tweetsearch.py`
 
-2. Or you can run it on a schedule with a cron task such as (insert the following into your Cron file):
+2. Or you can run it on a schedule with a cron task such as (insert the following into your Cron file using `crontab -e`):
 
-    `*/2 * * * * cd /path/to/home/github/spacex_smartweets && /path/to/python ./tweetsearch.py`
+    ```crontab
+    PATH=/usr/local/sbin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin:/path/to/home/.poetry/bin
+
+    */2 * * * * cd /path/to/home/github/spacex_smartweets && poetry run python tweetsearch.py
+    ```
+
+    Remember to have `/path/to/home/.poetry/bin`, and where `python3` resides in the PATH inside the crontab.
```

Comment:
In my current runtime, I use env...

`
*/2 * * * * . ~/python3.6/app/venv/bin/activate && python3 ~/github/spacex_smartweets/tweetsearch.py && deactivate
`

Something similar might help avoid the PATH declaration, if it could be easily explained.

Reply:
While `/path/to/home/.poetry/bin/poetry` could be called directly, I think it's reasonable to require that the python binary poetry is running under is required to be in `$PATH`.

Reply:
I'm not suggesting calling poetry via the explicit path directive. Rather, having a `venv` defined and activating it may avoid the possible problem entirely.

On my deploy system I have python 2.7 for one project, 3.6/3.9 for another, etc. ... so I use venv to avoid path clobbering.